config/vufind/contentsecuritypolicy.ini

```diff
@@ -84,3 +84,17 @@ base-uri[] = "'self'"
 ;max_age = 86400
 ; URL(s) for this reporting endpoint
 ;endpoints_url[] = 'https://example.report-uri.com'
+
+; Send the NEL (Network Error Logging) HTTP response header.
+; See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/NEL
+;[NetworkErrorLogging]
+; Set the named endpoint that borwsers use to report network errors.  The endpoint name
+; should match a group name in ReportTo above.
+;report_to = CSPReportingEndpoint
+; Maximum seconds to use this reporting endpoint.  Default (86400) is one day.
+;max_age = 86400
+; The following properties  are optional in the NEL specification, so VuFind will include
```

Comment:
Typo: extra space between "properties" and "are".

Reply:
Fixed